### PR TITLE
Use VARCHAR instead of CHAR for Databricks events table

### PR DESCRIPTION
**Pull request by istreeter**

### CODE REVIEW

File: docs/destinations/warehouses-and-lakes/rdb/loading-transformed-data/databricks-loader/index.md [hunk 1]
```diff
@@ -13,8 +13,7 @@ import CodeBlock from '@theme/CodeBlock';
 
 The following resources need to be created:
 
-- [AWS instance profile](https://docs.databricks.com/administration-guide/cloud-configurations/aws/instance-profiles.html) for giving permission to Databricks cluster to access S3 buckets securely
-- [Databricks cluster](https://docs.databricks.com/administration-guide/cloud-configurations/aws/instance-profiles.html#step-5-add-the-instance-profile-to-databricks) configured with the instance profile created above
+- [Databricks cluster](https://docs.databricks.com/clusters/create-cluster.html)
```
Comment (colmsnowplow) [inline docs/destinations/warehouses-and-lakes/rdb/loading-transformed-data/databricks-loader/index.md:16]: Checking that we deliberately removed 'aws iinstance profile' from the list, as it seems unrelated to the rest of the PR.

**Review by colmsnowplow — APPROVED** [review]
LGTM

**Review by voropaevp — APPROVED** [review]
Looks good. It would remove confusion, when values are autopadded.

File: docs/destinations/warehouses-and-lakes/rdb/loading-transformed-data/databricks-loader/index.md (hunk 1 above)
Comment (istreeter) [reply docs/destinations/warehouses-and-lakes/rdb/loading-transformed-data/databricks-loader/index.md:16]: It was deliberate, but I should have explained why in the PR.

Ever since RDB loader 4.2.0, the Databricks loader uses STS tokens for authentication, instead of requiring an instance profile.  From the user's point of view it is now much easier to set up the Snowplow/Databricks integration.

[Discourse announcement for 4.2.0](https://discourse.snowplow.io/t/rdb-loader-4-2-0-released/7151#new-authorization-options-for-snowflake-and-databricks-1).
